app/controllers/teachers/lessons_controller.rb

```diff
@@ -1,6 +1,16 @@
 module Teachers
   class LessonsController < BaseController
-    def show
+    before_action :load_resources, only: %i[show print]
+
+    def show; end
+
+    def print
```

Comment:
Minor (and not blocking the PR) point but in general we try to avoid non restful actions in controllers and just stick to the 7 standard actions and extract a new resource for this eg `PrintPreviewsController#show`.